gridstatus/nyiso.py

```diff
@@ -541,7 +543,7 @@ def get_interconnection_queue(self, verbose=False):
             "Project Name": "Project Name",
             "County": "County",
             "State": "State",
-            "Owner/Developer": "Interconnecting Entity",
+            "Developer Name": "Interconnecting Entity",
```

Comment:
In the raw data, only the "Interconnection Queue" tab has a "Developer Name" column. The "Owner/Developer" column was renamed in the `completed` dataframe but not the `withdrawn` dataframe.